packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -732,9 +749,16 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         return this.channel.applyStashedOp(innerContents.content);
     }
 
-    private verifyNotClosed() {
+    private verifyNotClosed(errorMessage: string, checkTombstone = true) {
         if (this._disposed) {
-            throw new Error("Context is closed");
+            throw new Error(`Context was closed during call of ${errorMessage}!`);
```

Comment:
I would not say "was closed / tombstoned during call of". It gives an impression that the data store was actually closed _during_ the call itself which can be confusing.
This should say something like "Context is closed. Error: ${errorMessage}"